app/actions.js

```diff
@@ -496,20 +513,26 @@ export const addChannel = ({ addr, channel }) => (dispatch, getState) => {
 }
 
 export const processLine = ({ message, addr }) => dispatch => {
+  const channel = message.content.channel
+  const cabalDetails = client.getDetails(addr)
+  const users = cabalDetails.getUsers()
+  const pmChannels = cabalDetails.getPrivateMessageList()
+  const isNewPmChannel = users[channel] && !pmChannels.includes(channel)
+
   const text = message.content.text
   if (text?.startsWith('/')) {
     const cabal = client.getCurrentCabal()
     cabal.processLine(text)
   } else {
-    dispatch(addMessage({ message, addr }))
+    if (isNewPmChannel) {
```

Comment:
if this is what i think it is (creating a new pm channel from an opened PM that hasn't been posted in until this message) let's add a comment?

```
// this creates the channel by sending a private message (PMs are not initiated until posting a message)